gclient/src/api/storage/mod.rs

```diff
@@ -58,6 +58,13 @@ impl GearApi {
         Ok(data.map(|(m, i)| (m.into(), i)))
     }
 
+    /// Get up to `count` messages from the mailbox.
+    pub async fn get_all_mailbox(&self, count: u32) -> Result<Vec<(StoredMessage, Interval<u32>)>> {
```

Comment:
Need to review all function names, as they looks inconsistent now. As an option:

- `get_from_mailbox` -> `get_mailbox_message`
- `get_from_account_mailbox` -> `get_account_mailbox_message`
- `get_all_mailbox` -> `get_mailbox_messages`